js/bybit.js

```diff
@@ -63,6 +63,7 @@ module.exports = class bybit extends Exchange {
                 'fetchOrderBook': true,
                 'fetchOrders': true,
                 'fetchOrderTrades': true,
+                'fetchPosition': true,
                 'fetchPositions': true,
                 'fetchPremiumIndexOHLCV': true,
                 'fetchTicker': true,
```

Comment:
@sc0Vu aren't we missing `fetchTicker`upgrade? 
I'm not seeing this endpoint being used here: https://bybit-exchange.github.io/docs/derivativesV3/contract/#t-dv_tickerhead
![image](https://user-images.githubusercontent.com/43336371/201648012-ec72f3f5-29f0-430a-ba51-d195985f638d.png)